modules.json

```diff
@@ -59,5 +59,10 @@
     "tabs": [
       "SoundMatrix"
     ]
+  },
+  "unittest": {
+    "tabs": [
+      "UnitTest"
+    ]
```

Comment:
Is there a reason we decided to name the module `unittest`? Don't think its an issue but I'm just thinking maybe just `test` is more straightforward and less confusing for our fellow freshmen.

Reply:
I'll rename it into `testing`